### ISSUE TITLE
A11Y Lab - Contrast (Minimum)

### ISSUE THREAD

**Issue by glenn-sorrentino** [scidsg/hushline#159]
Contrast (Minimum): The visual presentation of text and images of
text has a contrast ratio of at least 4.5:1, except for the following:
• Large Text: Large-scale text and images of large-scale text have a contrast ratio
of at least 3:1;
• Incidental: Text or images of text that are part of an inactive user interface
component, that are pure decoration, that are not visible to anyone, or that are
part of a picture that contains significant other visual content, have no contrast
requirement.
• Logotypes: Text that is part of a logo or brand name has no minimum contrast
requirement.
The intent of this Success Criterion is to provide enough contrast between text and its
background so that it can be read by people with moderately low vision (who do not use
contrast-enhancing assistive technology).
Color deficiencies can affect luminance contrast somewhat. Therefore, in the
recommendation, the contrast is calculated in such a way that color is not a key factor
so that people who have a color vision deficit will also have adequate contrast between
the text and the background.
You can use one of these tools to validate any contrast.

**Comment by glenn-sorrentino** [comment]
https://github.com/scidsg/project-info/blob/main/hush-line/1.%20Accessibility/WCAG%202.1%20AAA_HushLine%20website.pdf

**Comment by glenn-sorrentino** [comment]
👍 Fixed all contrast issues

Confirmed on a11y.com
![Screenshot 2023-10-19 at 12 31 59 PM](https://github.com/scidsg/hushline/assets/28545431/314a92dd-d603-4a03-8844-ec4a04f50375)
